out-adapter-persistence-jpa/src/test/java/gohigher/application/ApplicationPersistenceQueryAdapterTest.java

```diff
@@ -35,6 +36,58 @@ void setUp() {
 		applicationPersistenceQueryAdapter = new ApplicationPersistenceQueryAdapter(applicationRepository);
 	}
 
+	@DisplayName("findByIdAndUserId 메서드는")
+	@Nested
+	class Describe_findByIdAndUserId {
+
+		private final long userId = 1L;
+
+		@DisplayName("존재하는 지원서 id로 조회하면")
+		@Nested
+		class Context_with_exist_application_id {
+
+			private final long applicationId = 1L;
+
+			@BeforeEach
+			void setUp() {
+				ApplicationJpaEntity naverApplication = convertToApplicationEntity(userId,
+					NAVER_APPLICATION.toDomain());
+				given(applicationRepository.findByIdAndUserIdWithProcess(applicationId, userId))
+					.willReturn(Optional.ofNullable(naverApplication));
```

Comment:
```suggestion
				given(applicationRepository.findByIdAndUserIdWithProcess(applicationId, userId))
					.willReturn(Optional.of(naverApplication));
```
해당 ApplicationJpaEntity 는 null 이 될 가능성이 없으니 of 로 받아도 될것같습니다.

Reply:
수정했습니다!